src/axom/mint/execution/internal/structured_exec.hpp

```diff
@@ -97,11 +97,11 @@ struct structured_exec< CUDA_EXEC< BLOCK_SIZE, SYNCHRONOUS > >
   /* *INDENT-OFF* */
 
   using loop2d_policy = RAJA::KernelPolicy<
-    RAJA::statement::CudaKernel<
+    RAJA::statement::CudaKernelFixed< 256,
```

Comment:
The only thing I would suggest here is not hard coding the block size and tile dimensions as numerical values but rather using a named variable, macro, etc. (whatever makes sense) to be able to more easily adjust these values by making a one-line change.

Reply:
Yes -- that makes sense.

Reply:
Done.